gui/wxpython/lmgr/giface.py

```diff
@@ -44,7 +44,7 @@ def __dir__(self):
         return self._pydata[0].keys()
 
     def __str__(self):
-        return '' if self.maplayer.name is None else self.maplayer.name
+        return '' if self.maplayer is None else self.maplayer.name
```

Comment:
I think it would be good to keep the check for name there as well:
```suggestion
        return '' if (self.maplayer is None or self.maplayer.name is None) else self.maplayer.name
```